src/clib/xtg/surf_get_z_from_xy.c

```diff
@@ -79,13 +79,15 @@ surf_get_z_from_xy(double x,
         return UNDEF;
     }
 
-    /* two approaches here; the 'userelative' is more clean and safe? */
+    /* two approaches here; the 'userelative' option = 1 is more clean and safe? */
 
-    if (userelative == 1) {
-        /* map origin rleative is 0.0 */
+    if (option == 0) {
```

Comment:
possible to use constants instead of magic numbers in order to make the code more readable?

Reply:
That may be possible with C macros, will consider (but no high priority)